modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1225,6 +1225,40 @@ definition {
 		AppBuilderAdmin.deleteEntry(itemName = "My Form View 1");
 	}
 
+	@description = "Delete a publishing"
+	@priority = "5"
+	test DeletePublishing {
+		property portal.acceptance = "false";
```

Comment:
hey @nkkwon , the test is okay! could you please just remove this annotation related to acceptance test? Thanks!